dashboard/app/reporting_email.go

```diff
@@ -179,16 +179,22 @@ func emailSendBugNotif(c context.Context, notif *dashapi.BugNotification) error
 	if err := json.Unmarshal(notif.Config, cfg); err != nil {
 		return fmt.Errorf("failed to unmarshal email config: %v", err)
 	}
-	to := email.MergeEmailLists([]string{cfg.Email}, notif.CC)
+	to := []string{cfg.Email}
+	cc := []string{}
 	if cfg.MailMaintainers && notif.Public {
 		to = email.MergeEmailLists(to, notif.Maintainers, cfg.DefaultMaintainers)
 	}
+	if len(to) < 1 {
```

Comment:
I would prefer `if len(to) == 0 {` for a check for emptiness.
However, we initialized to with cfg.Email, so I don't see how it can be potentially empty here...